src/hooks/useDivisionOptions.tsx

```diff
@@ -10,15 +14,20 @@ type DivisionOption = {
 const useDivisionOptions = (): DivisionOption[] => {
   const locale = useLocale();
   const { data: neighborhoodsData } = useNeighborhoodListQuery();
+  const neighborhoodList = neighborhoodsData?.neighborhoodList.data;
+  const filteredNeighborhoodOptionList = neighborhoodList?.filter((option) =>
+    DIVISION_BLOCKLIST.every((blockOption) => blockOption !== option.id)
+  );
+
+  const neighborhoodToOptionValue = (neighborhood: Neighborhood) => ({
+    text: getLocalisedString(neighborhood.name, locale),
+    value: neighborhood.id,
+  });
```

Comment:
I think it is needless here to create a new variable that is used only once with a map function.

Reply:
Purpose is to only make it more readable, but maybe going too far?

Reply:
Updated :)